v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -346,57 +269,42 @@ Before running the benchmark, it's important to allocate partitions to workload
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ ulimit -n 200500 && ./workload run tpcc \
-    --partitions 27 \
-    --warehouses 100000 \
-    --partition-affinity 0,5,10,15,20,25 \
-    --ramp 30m \
-    --duration 1ms \
-    $(cat addrs)
+    ulimit -n 500500 && ./workload run tpcc --partitions 81 \
+    --warehouses 150000 --active-warehouses $NUMWARE \
+    --partition-affinity 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16 \
+    --ramp 30m --duration 1ms --histograms workload1.histogram.ndjson $(cat addrs)
     ~~~
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ ulimit -n 200500 && ./workload run tpcc \
-    --partitions 27 \
-    --warehouses 100000 \
-    --partition-affinity 1,6,11,16,21,26 \
-    --ramp 30m \
-    --duration 1ms \
-    $(cat addrs)
+    ulimit -n 500500 && ./workload run tpcc --partitions 81 \
+    --warehouses 150000 --active-warehouses $NUMWARE \
```

Comment:
Obv. I will have the same question re: `$NUMWARE` for all these commands.

Are they each the same value?  Or do they differ?